src/callback.rs

```diff
@@ -131,4 +137,9 @@ pub(crate) mod test_util {
             }
         }
     }
+
+    #[derive(Default)]
+    struct DefaultCallbackProperties {
```

Comment:
What's this for?

Reply:
An older version of a test to check whether one can create a Properties object with this. But it is not needed now. I will remove it.

Reply:
Hey @TheNeikos were you still planning to remove this? I'd be happy to merge this PR after :)

Reply:
Yes! Sorry, it dropped off my radar.